packages/components/pie-modal/src/index.ts

```diff
@@ -43,27 +43,23 @@ export class PieModal extends RtlMixin(LitElement) implements ModalProps {
     @validPropertyValues(componentSelector, headingLevels, 'h2')
     public headingLevel: ModalProps['headingLevel'] = 'h2';
 
-    @property({ type: Boolean, reflect: true })
-    public isDismissible = false;
-
     @property({ type: Boolean })
     public hasBackButton = false;
 
     @property({ type: Boolean })
-    public isFullWidthBelowMid = false;
+    public hasStackedActions = false;
```

Comment:
I think you'll need to add this to the `defs` file too.

Reply:
Fixed

Reply:
I'd recommend updating `defs` first when adding new properties. That will then ensure you have to update SB and the component together thanks to typescript :)